### DISCUSSION TITLE
Baseline file changes / sorting

### DISCUSSION THREAD

**Discussion by Zeckie** [KotlinIsland/basedmypy#188] [Ideas] [answered]
I've noticed that small changes in source code sometimes result in large changes to the `baseline.json` file (when run with `--write-baseline`).

It seems to be that the source files within the baseline aren't sorted, and sometimes the order changes.

Are the errors in the same source file written to the baseline in a specific order?

I think they should be sorted (both the keys of the json dict, using `sort_keys=True`, and the lists of errors)

https://github.com/KotlinIsland/basedmypy/blob/master/mypy/errors.py#L799

**Comment by KotlinIsland** [comment, marked as answer]
Implemented in #212
